workspaces/build.rs

```diff
@@ -5,6 +5,8 @@ fn main() {
     let env_bin = std::env::var("NEAR_SANDBOX_BIN_PATH").is_ok();
     if !doc_build && !env_bin && cfg!(feature = "install") {
         // TODO Update commit to stable version once binaries are published correctly
-        near_sandbox_utils::install().expect("Could not install sandbox");
+        // Commit: https://github.com/near/nearcore/commit/dce4c514f60a7f7981f00290268af4faf7eec55c
+        near_sandbox_utils::install_with_version("master/eb2bbe1c3f51912c04462ce988aa496fab03d60e")
+            .expect("Could not install sandbox");
```

Comment:
Please, update near_sandbox_utils to use the most recent stable release of nearcore.

Reply:
fixed